aruba/features/step_definitions/endpoints.rb

```diff
@@ -15,7 +15,9 @@
              " -t '#{ENV['TPP_ACCESS_TOKEN']}'" +
              " -insecure",
 
-    "Cloud" => "-u '#{ENV['CLOUD_URL']}' -k '#{ENV['CLOUD_APIKEY']}'"
+    "Cloud" => "-u '#{ENV['CLOUD_URL']}' -k '#{ENV['CLOUD_APIKEY']}'",
```

Comment:
is it not possible to reuse the expression instead of to duplicate it?

Reply:
Already resolved in next uploaded commit